superset-frontend/src/explore/components/DatasourcePanel/index.tsx

```diff
@@ -97,26 +101,55 @@ const DatasourceContainer = styled.div`
 `;
 
 const LabelWrapper = styled.div`
-  overflow: hidden;
-  text-overflow: ellipsis;
+  ${({ theme }) => css`
+    overflow: hidden;
+    text-overflow: ellipsis;
+    font-size: ${theme.typography.sizes.s}px;
+    background-color: ${theme.colors.grayscale.light4};
+    margin: ${theme.gridUnit * 2}px 0;
+    border-radius: 4px;
```

Comment:
```suggestion
    border-radius: ${theme.gridUnit}px;
```

Reply:
I don't think that is related to gridUnit. Supposedly, if we created a theme where distances between components are bigger, that doesn't mean we want the corners to be more rounded